app/views/site/how_it_works_newbie.erb

```diff
@@ -8,55 +8,51 @@
     <div class="col-md-8">
 
       <p>
-      Exercism is spread across two locations:
+        Using Exercism requires three tools:
       </p>
 
       <ol>
-        <li><b>Your local programming environment</b>: write a solution to the exercise using your text editor, not a browser.</li>
-        <li><b>The <a href="http://exercism.io">exercism.io</a> website</b>: submit the solution to the site, so that you can get feedback and discuss it with other learners.</li>
+        <li><strong>Your text editor</strong>: Write a solution to an exercise using your favorite text editor.</li>
+        <li><strong>Your command line interface</strong>: Fetch problems and submit solutions via the command line (or terminal).</li>
+        <li><strong>The <a href="http://exercism.io">exercism.io</a> website</strong>: Review feedback on your solution and discuss it with other learners on the website.</li>
       </ol>
 
       <p>
       For each exercise that you do, you'll go through the same basic steps.
       </p>
 
       <ol>
-        <li>Download the exercise.</li>
-        <li>Write code to solve the exercise on your own computer.</li>
-        <li>Submit your solution to the site so that you can get feedback on it.</li>
-        <li>Go look at how other people solved the same problem, and ask questions about what seems interesting or confusing.</li>
+        <li><strong>Fetch the exercise</strong> using the command line.</li>
+        <li><strong>Write code</strong> to solve the exercise on your own computer, satisfying each of the tests.</li>
+        <li><strong>Submit your solution</strong> using the command line.</li>
+        <li><strong>Review feedback</strong> and look at how other people solved the same problem on the website. Ask questions about what seems interesting or confusing!</li>
+        <li><strong>Improve your solution</strong> and resubmit as many times as desired.</li>
```

Comment:
The user literally types "submit" again which to me is more clear with the word "resubmit" but I am open to more ideas on the best way to describe this in English. I don't think the alternate you provided makes it more clear. At least for me, it sounds more confusing. 

Additionally, I don't want to create a very long description here as the purpose of the section is only give potential users a better understanding of what exercism is about. Perhaps further down the page where the instructions are detailed is a better place to expound on the full process of submission and iteration. More detail is already there, but we could also add that they can see each individual iteration on the website if you think that would improve the instructions.